static/app/views/preprod/buildDetails/sidebar/buildDetailsSidebarContent.tsx

```diff
@@ -122,13 +123,15 @@ export function BuildDetailsSidebarContent(props: BuildDetailsSidebarContentProp
 
   return (
     <Flex direction="column" gap="2xl">
-      {/* App info */}
-      <BuildDetailsSidebarAppInfo
-        appInfo={buildDetailsData.app_info}
-        sizeInfo={buildDetailsData.size_info}
-        projectId={props.projectId}
-        artifactId={props.artifactId}
-      />
+      {/* App info - only show when artifact is processed */}
+      {buildDetailsData.state === BuildDetailsState.PROCESSED && (
```

Comment:
Do we currently also show the processing state when the whole build is processing?